src/components/HomePage/FAQSection.js

```diff
@@ -137,16 +137,16 @@ function FAQSection() {
 				vibe best with.`
 		},
 		{
-			question: `When is HOTH 9 taking place? How long is it going to be?`,
-			answer: `The hackathon will take place on Saturday, February 26, 2022, 
+			question: `When is HOTH X taking place? How long is it going to be?`,
+			answer: `The hackathon will take place on TBD 
 				and it will be 12 hours long.`
 		},
 		{
-			question: `Where is HOTH 9 taking place?`,
+			question: `Where is HOTH X taking place?`,
 			answer:
 				<>
-					Currently, we are planning for HOTH 9 to be hybrid
-					with the in-person portion of the event taking place at Covel Grand Horizons. A capped number
+					Currently, we are planning for HOTH X to be in-person
+					with the event taking place at TBD. A capped number
 					of people can participate in-person, and all in-person attendees must provide
```

Comment:
Could you take out the rest of the answer after "the event taking place at TBD" since that part talks about the hybrid format and we're doing fully in-person?

Reply:
Just fixed